src/runtimes/standalone.ts

```diff
@@ -193,7 +204,7 @@ export const standalone = (props: RuntimeProps) => {
         const credentialsProvider: CredentialsProvider = auth.getCredentialsProvider()
 
         // Initialize every Server
-        const disposables = props.servers.map(s => s({ credentialsProvider, lsp, workspace, telemetry, logging }))
+        const disposables = props.servers.map(s => s({ chat, credentialsProvider, lsp, workspace, telemetry, logging }))
```

Comment:
This is a breaking change for servers, right?

Reply:
yes